src/components/card/CardSidebar.vue

```diff
@@ -127,6 +127,11 @@
 					href="https://deck.readthedocs.io/en/latest/Markdown/"
 					target="_blank"
 					class="icon icon-info" />
+				<Actions v-if="canEdit">
+					<ActionButton v-if="descriptionEditing" icon="icon-attach" @click="showAttachmentModal()">
```

Comment:
Something is wrong with the sizing of that icon make sure that the background iamge is only 16x16px.